test/embedded/classes-wasm.swift

```diff
@@ -0,0 +1,35 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+// RUN: %swift-frontend -c %t/check.swift -parse-as-library -target wasm32-unknown-none-wasm \
+// RUN:   -enable-experimental-feature Embedded -Xcc -fdeclspec -disable-stack-protector \
+// RUN:   -o %t/check.o
+// RUN: %clang -target wasm32-unknown-none-wasm %t/check.o %t/rt.c -nostdlib -o %t/check.wasm
+// RUN: %target-run %t/check.wasm
+// REQUIRES: executable_test
+// REQUIRES: CPU=wasm32
```

Comment:
Can you point me to a CI job that actually runs these executable WASM tests?

Reply:
Wasm CI runs executable tests on WasmKit runtime: https://ci.swift.org/job/swift-PR-Linux-crosscompile-wasm/31